AsyncDisplayKit/Details/ASCollectionViewFlowLayoutInspector.h

```diff
@@ -47,12 +50,37 @@
  */
 - (void)didChangeCollectionViewDelegate:(id<ASCollectionDelegate>)delegate;
 
+/**
+ * Allow the inspector to respond to dataSource changes.
+ *
+ * @discussion A great time to update perform selector caches!
+ */
+- (void)didChangeCollectionViewDataSource:(id<ASCollectionDataSource>)dataSource;
+
 @end
 
+/**
+ * A layout inspector for non-flow layouts that returns a constrained size to let the cells layout itself as
+ * far as possible based on the scrollable direction of the collection view. It throws exceptions for delegate
+ * methods that are related to supplementary node's management.
+ */
+@interface ASCollectionViewDefaultCustomLayoutInspector : NSObject <ASCollectionViewLayoutInspecting>
```

Comment:
I think we should go with `ASCollectionViewLayoutInspector`. Just by reading `ASDefaultLayoutInspector` I would not really figure out why it's there or for what I would use it.